legacy/src/api/auth.md

```diff
@@ -22,6 +22,10 @@ API calls can be authenticated by either providing an API key in the
 "X-Api-Key" header or by providing a user access token in the "Authorization"
 header.
 
+If you are using user access token for authentication and accessing/creating org
+[account][] resources then "X-Centrapay-Account" header must be provided with the org
+account's id.
```

Comment:
I think this could be reworded a bit. Maybe...

Org Accounts accessed with a user access token require the "X-Centrapay-Account" header to be provided. The "X-Centrapay-Account" header specifies the unique identifier of the Centrapay Org Account.